server/views/applications/check-your-answers.njk

```diff
@@ -0,0 +1,134 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "../partials/layout.njk" %}
+
+{% block pageTitle %}
+ Approved Premises - Check your answers
```

Comment:
We've been using `{% set pageTitle = applicationName + " - " + pageHeading  %}` in the other views
We tend to `{% set mainClasses = "app-container govuk-body" %}` elsewhere but that might not work with this page